### PR TITLE
Add NIP checklist

### PR DESCRIPTION
**Pull request by shaibearary**

### CODE REVIEW

**Comment by shaibearary** [comment]
@BlowaterNostr

File: readme.md [hunk 1]
```diff
@@ -21,6 +21,50 @@ globally available decentralized chat experience.
...
+- [ ] NIP-09 - Event Deletion
+- [x] NIP-10 - Conventions for clients' use of e and p tags in text events
+- [ ] NIP-11 - Relay Information Document
+- [x] NIP-12 - Generic Tag Queries
+- [ ] NIP-13 - Proof of Work
+- [ ] NIP-14 - Subject tag in text events
+- [ ] NIP-15 - Nostr Marketplace (for resilient marketplaces)
+- [ ] NIP-16 - Event Treatment
+- [ ] NIP-18 - Reposts
+- [x] NIP-19 - bech32-encoded entities
```
Comment (BlowaterNostr) [inline readme.md:43]: partial

File: readme.md [hunk 2]
```diff
@@ -21,6 +21,50 @@ globally available decentralized chat experience.
...
+#### nostr features supported
+
+- [x] NIP-01 - Basic protocol flow description
+- [ ] NIP-02 - Contact List and Petnames
+- [ ] NIP-03 - OpenTimestamps Attestations for Events 
+- [x] NIP-04 - Encrypted Direct Message
+- [ ] NIP-05 - Mapping Nostr keys to DNS-based internet identifiers
+- [ ] NIP-06 - Basic key derivation from mnemonic seed phrase
+- [x] NIP-07 - window.nostr capability for web browsers
+- [x] NIP-08 - Handling Mentions
```
Comment (BlowaterNostr) [inline readme.md:33]: not sure

**Comment by BlowaterNostr** [comment]
@shaibearary unless we have acceptance test for NIPs, otherwise it's always just a best effort coordination.

**Pull request closed without merging**